packages/agenda/examples-todo/src/agenda/agenda.routes.ts

```diff
@@ -3,9 +3,9 @@ import { Routes } from '@angular/router';
 import { AgendaDemoPageComponent } from './pages/demo/demo.page';
 
 export const AGENDA_EXAMPLES_ROUTES: Routes = [
-	// {
-	// 	path: '',
-	// 	component: AgendaDemoPageComponent,
-	// 	pathMatch: 'full',
-	// },
```

Comment:
🤔

Reply:
Ik had de example eerst proberen verbergen door de route in comments te zetten. Maar dat was niet de juiste oplossing. Dus heb de example folder van de agenda tijdelijk een andere naam gegeven.